modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -3190,15 +3195,15 @@
             return s.join("&").replace(/%20/g, "+");
         },
 
-        storage: function () {
+        storage: (function () {
```

Comment:
`atmosphere.util.storage` is being checked as a boolean and is being assigned boolean values. Declaring it as a function looked like a mistake to me.